src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/PropertyDescriptorCollection.cs

```diff
@@ -6,6 +6,7 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 
+// TODO-NULLABLE: This class needs more attention. Many APIs allow null properties but some other don't expect it (i.e. Find).
```

Comment:
Are you going to handle that in this PR?

Reply:
no, this requires product changes